### ISSUE TITLE
Super robot wars V and X crash

### ISSUE THREAD

**Issue by Lynle23** [Vita3K/Vita3K-Android#460]
Those games keep on crashing during character's dialogue and when new enemies appear on the map, especially from scenario 6 onward.

**Comment by Geekumag** [comment]
I agree bro its stuck when new enemies appear and quit to main menu of my phone and overheat alot i thought because of i modified save cheat money and tac and parts 🤣

**Comment by zr753** [comment]
Same issue here.
Device: SnapDragon 865(Retroid Pocket 5)
OS: Android 13
Memory: 8 GB RAM
Game: [PCSH10004] Super Robot War V

Setting: Vulkan, 2x resolution (1920*1088)
Driver: Turnip_v26.0.0_R2_Gmem

Issue: Game crashes after chapter 14 when: choosing weapon, enemy appears, finish round etc.

**Comment by gship408-coder** [comment]
Using exactly the same thing as you and stuck in episode 6

**Comment by Jcoezz** [comment]
Got the same problem, stuck at scenario 6 at SRW X. Its running smoothly even on default settings but keep crashing when new enemy appears.
